tck/src/main/java/org/eclipse/sparkplug/tck/test/common/Utils.java

```diff
@@ -237,6 +237,12 @@ private enum HostState {
 		OFFLINE
 	}
 
+	public enum Profile {
```

Comment:
It seems to me that enums are constants and would fit better in the Constants class? (Yes it's named TopicConstants at the moment but we could rename it to just Constants. It already contains constants that are not topics).

Reply:
Actually I could do this in another PR.

Reply:
yes pls. do this in another PR after this one

Reply:
will do